vNext/src/ClientGenerator/BootstrapCodegen.proj

```diff
@@ -0,0 +1,48 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@jdom created this project to clean up the way that the bootstrap versions of projects were built. These directives previously existed in `Orleans.csproj`, but were moved here largely for cleanliness